<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rezult_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiplication">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_one">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_two">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="rezult_save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	qproperty-alignment: 'AlignCenter';
	border: 1px solid gray;
}

background-color:white;</string>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
